redbot/setup.py

```diff
@@ -262,13 +268,13 @@ async def remove_instance(
     if _create_backup is True:
         await create_backup(instance)
 
-    backend = get_current_backend(instance)
-    driver_cls = drivers.get_driver_class(backend)
-    await driver_cls.initialize(**data_manager.storage_details())
-    try:
-        if delete_data is True:
+    driver_cls = drivers.get_driver_class(backend)  
```

Comment:
The code can't have trailing whitespace:
```suggestion
    driver_cls = drivers.get_driver_class(backend)
```